streams/src/main/java/org/apache/kafka/streams/StreamsConfig.java

```diff
@@ -509,6 +514,12 @@
                     "",
                     Importance.MEDIUM,
                     CLIENT_ID_DOC)
+            .define(CLOSE_WAIT_MS_CONFIG,
```

Comment:
Do we need this new config? According to the ticket, we want to used `KafkaStreams#close(Duration timeout)`.